GitExtUtils/GitUI/Theming/AppColorDefaults.cs

```diff
@@ -24,11 +24,45 @@ public static class AppColorDefaults
                 { AppColor.DiffAddedExtra, Color.FromArgb(135, 255, 135) }
             };
 
-        public static Color GetBy(AppColor name)
+        private static readonly Dictionary<string, Dictionary<AppColor, Color>> Variations = new()
         {
-            return Values.TryGetValue(name, out var result)
-                ? result
-                : FallbackColor;
+            {
+                "colorblind", new()
+                {
+                   { AppColor.Graph, Color.FromArgb(0x06, 0x00, 0xa8) },
```

Comment:
Question: The invarientTheme is read from invariant.css, these are hardcoded in the app.
Is there a reason for this?
I would expect the invariant theme to be hardcoded, but it is nice to have the colors listed in a file too.
(Still hoping themes can return...)

Reply:
I agree, I don't  like having both hardcoded colors and .css colors either. Would rather keep everything in .css for simplicity